packages/indy-vdr/src/anoncreds/IndyVdrAnonCredsRegistry.ts

```diff
@@ -218,20 +219,26 @@ export class IndyVdrAnonCredsRegistry implements AnonCredsRegistry {
 
       const response = await pool.submitReadRequest(request)
 
+      //FIXME: Is the ledgerType 1 ok or is it meant to be a dynamic value?
+      // if yes, what the best way to get the ledgerType
+      const schema = await this.fetchIndySchemaWithSeqNo(agentContext, response.result.ref, did, 1)
```

Comment:
When fetching a schema it will always be `1`, which is the `DOMAIN` ledger.  If you look at e.g. https://indyscan.io/home/SOVRIN_MAINNET, you can see there's three subledgers.

So I think you can just hardcode the `1` inside the `fetchIndySchemaWithSeqNo` method as it will always be 1